### PR TITLE
[GSW-POOL] fix: Fix entire comment pool detail

### PR DESCRIPTION
**Pull request by khiemldk**

### CODE REVIEW

File: packages/web/src/components/pool/my-position-card/MyPositionCardClaimContent.tsx [hunk 1]
```diff
@@ -33,54 +36,60 @@ export const MyPositionClaimContent: React.FC<MyPositionClaimContentProps> = ({
...
-  const externalRewards = useMemo(() => {
-    if (!rewardInfo) {
-      return null;
-    }
-    if (rewardInfo.EXTERNAL.length === 0) {
-      return null;
-    }
-    return rewardInfo.EXTERNAL;
-  }, [rewardInfo]);
+  // const externalRewards = useMemo(() => {
```
Comment (jinoosss) [inline packages/web/src/components/pool/my-position-card/MyPositionCardClaimContent.tsx:39]: Remove unused lines.

File: packages/web/src/components/pool/my-position-card/MyPositionCardRewardContent.tsx [hunk 2]
```diff
@@ -24,27 +26,27 @@ export const MyPositionRewardContent: React.FC<MyPositionRewardContentProps> = (
     return rewardInfo.STAKING;
   }, [rewardInfo.STAKING]);
 
-  const externalRewards = useMemo(() => {
-    if (rewardInfo.EXTERNAL.length === 0) {
-      return null;
-    }
-    return rewardInfo.EXTERNAL;
-  }, [rewardInfo.EXTERNAL]);
+  // const externalRewards = useMemo(() => {
```
Comment (jinoosss) [inline packages/web/src/components/pool/my-position-card/MyPositionCardRewardContent.tsx:29]: Remove unused lines.

File: packages/web/src/pages/earn/pool/[pool-path]/add.tsx [hunk 3]
```diff
@@ -25,15 +24,15 @@ export default function EarnAdd() {
       { title: "Earn", path: "/earn" },
       {
         title:
-          breakpoint === DEVICE_TYPE.WEB
+          width >= 1250
```
Comment (jinoosss) [inline packages/web/src/pages/earn/pool/[pool-path]/add.tsx:27]: Declare a variable corresponding to 1250.

File: packages/web/src/pages/earn/pool/[pool-path]/remove.tsx [hunk 4]
```diff
@@ -24,15 +23,15 @@ export default function Earn() {
       { title: "Earn", path: "/earn" },
       {
         title:
-          breakpoint === DEVICE_TYPE.WEB
+        width >= 1150
```
Comment (jinoosss) [inline packages/web/src/pages/earn/pool/[pool-path]/remove.tsx:26]: Declare a variable corresponding to 1150.

**Review by jinoosss — DISMISSED** [review]
I left small comments.

File: packages/web/src/pages/earn/pool/[pool-path]/remove.tsx (hunk 4 above)
Comment (khiemldk) [reply packages/web/src/pages/earn/pool/[pool-path]/remove.tsx]: done

File: packages/web/src/pages/earn/pool/[pool-path]/add.tsx (hunk 3 above)
Comment (khiemldk) [reply packages/web/src/pages/earn/pool/[pool-path]/add.tsx]: done

File: packages/web/src/components/pool/my-position-card/MyPositionCardRewardContent.tsx (hunk 2 above)
Comment (khiemldk) [reply packages/web/src/components/pool/my-position-card/MyPositionCardRewardContent.tsx]: done

File: packages/web/src/components/pool/my-position-card/MyPositionCardClaimContent.tsx (hunk 1 above)
Comment (khiemldk) [reply packages/web/src/components/pool/my-position-card/MyPositionCardClaimContent.tsx]: done

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**